src/psyclone/tests/gocean0p1_test.py

```diff
@@ -1,18 +1,23 @@
 # ----------------------------------------------------------------------------
-# (c) Science and Technology Facilities Council, 2016
+# (c) Science and Technology Facilities Council, 2016-2019
```

Comment:
Please update this copyright/license info.

Reply:
Done.